src/main/java/wedlog/address/logic/parser/VendorAddCommandParser.java

```diff
@@ -42,14 +44,12 @@ public VendorAddCommand parse(String args) throws ParseException {
                 : ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = null;
         Address address = null;
-        Set<Tag> tagList = null;
```

Comment:
This comment is added here because I [cannot comment on unchanged lines](https://github.com/github/roadmap/issues/347).

Should `ArgumentTokenizer` tokenize `PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_TAG` too? Is `email` and `address` supposed to be null for all vendors?